addons/remove-search-bar-autocomplete/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Remove autocomplete from search bar",
+  "description": "Stops web browsers from autocompleting suggestions in the navigation bar's search bar.",
+  "tags": ["community"],
+  "credits": [
+    {
+      "name": "jbthepig",
+      "link": "https://scratch.mit.edu/users/jbthepig"
+    }
+  ],
+  "dynamicEnable": true,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/*"]
```

Comment:
I recommend adding `"runAtComplete": false` underneath `matches` just so it starts running a tad bit faster.